### DISCUSSION TITLE
Triple difference

### DISCUSSION THREAD

**Discussion by abshev** [bcallaway11/did#181] [Q&A]
I'm running a triple difference analysis with staggered treatments.  It is straightforward to obtain point estimates for triple difference estimators; however, it is not clear to me how to use the did package for inference in this setting.  I found some [slides](https://psantanna.com/files/DDD.pdf) from Dr. Sant'anna that suggest making use of the influence functions and the multiplier bootstrap for this purpose, but the details aren't given.  I was wondering if you could shed some light on how one might do this with the did package.

**Comment by raffaelemancuso** [comment]
Hello,

> It is straightforward to obtain point estimates for triple difference estimators;

It is not so straightforward to me how  to obtain point estimates for triple difference estimators

Would you mind explaining to me how you did it?
